### PR TITLE
I318 Setup nightly build tests

### PR DESCRIPTION
**Pull request by Tereza-Pytelova**

### CODE REVIEW

File: ci/unit_tests/workspace_delete/main_test.py [hunk 1]
```diff
@@ -71,6 +71,7 @@ def _deleteAllWorkspaces(self):
 
         assert len(workspaces) == 0
 
+    @pytest.mark.skipif(os.environ.get('TRAVIS_EVENT_TYPE') != "cron", reason="This test is nightly build only.")
     @pytest.mark.parametrize('envVarNameUsername, envVarNamePassword', [('WA_USERNAME', 'WA_PASSWORD')])
     def test_deleteById(self, envVarNameUsername, envVarNamePassword):
```
Comment (ota-sandr) [inline ci/unit_tests/workspace_delete/main_test.py]: I think that this test should be enabled for non-nightly builds

File: ci/unit_tests/functions_deploy/main_test.py [hunk 2]
```diff
@@ -109,6 +109,7 @@ def test_functionsUploadFromDirectory(self, useApikey):
             functionRespJson = functionResp.json()
             assert "Hello unit test!" in functionRespJson['greeting']
 
+    @pytest.mark.skipif(os.environ.get('TRAVIS_EVENT_TYPE') != "cron", reason="This test is nightly build only.")
     def test_pythonVersionFunctions(self):
```
Comment (ota-sandr) [inline ci/unit_tests/functions_deploy/main_test.py]: Maybe we can add `@pytest.mark.parametrize('useApikey', [True, False])` to this test and to `test_functionsInZip`

Comment (Tereza-Pytelova) [reply ci/unit_tests/functions_deploy/main_test.py]: Added.

File: ci/unit_tests/workspace_delete/main_test.py (hunk 1 above)
Comment (Tereza-Pytelova) [reply ci/unit_tests/workspace_delete/main_test.py]: Yes, I forgot to save changes :)

**Pull request merged**